templates/studygroups/email/weekly-update.html

```diff
@@ -331,14 +331,18 @@ <h2>
       <thead>
         <tr>
           <td>{% trans "Name" %}</td>
+          {% if show_emails %}
```

Comment:
We're also including email addresses where we're showing facilitator names in the other tables. Search for the mailto: links in this template.